packages/components/psammead-section-label/src/index.test.jsx

```diff
@@ -236,4 +236,19 @@ describe('SectionLabel', () => {
       ).toEqual('section_name');
     });
   });
+
+  describe('With heading overriden to be a strong element', () => {
```

Comment:
Maybe a test case for an invalid override element?

Reply:
@HarveyPeachey [done! ](https://github.com/bbc/psammead/pull/3585/files#diff-00f9b2243883179eeed7e9ef2e9fe3b3R255-R272)